src/OrchardCore/OrchardCore.Localization.Core/DynamicData/DynamicDataTranslationsProvider.cs

```diff
@@ -0,0 +1,14 @@
+using System.Linq;
+
+namespace OrchardCore.Localization.DynamicData
+{
+    public class DynamicDataTranslationsProvider : ITranslationProvider
+    {
+        public void LoadTranslations(string cultureName, CultureDictionary dictionary)
+        {
+            // TODO: Load the translation from the database
```

Comment:
If you want an example, you could take a look at the TemplatesManager class that store a document of a specific type in the database. You will need one like this, I assume a single containing all supported cultures.

```csharp
public class TranslationsDocument
{
  public List<Translation> Translations { get; set; }
}

public class Translation
{
  public string Context { get; set; } // "Content Types - Article"
  public string Value { get; set; } // "Article"
  public Dictionary<string, string> Values { get; set; }
}
```

Where values will be:
```
{
  "en-US": "Article",
  "fr": "Article",
  "es" "Artículo"
}
```

The advantage with using a document like this is that we can use the exact same json format in the import/export step.

Reply:
I see, I was thinking to store them in the main recipe at the beginning

Looks intereseted I will have a look